internal/services/firewall/parse/firewall_policy_rule_collection.go

```diff
@@ -0,0 +1,83 @@
+package parse
+
+// NOTE: this file is generated via 'go:generate' - manual changes will be overwritten
```

Comment:
This file (and its test and the counterparts in the *validate* module) is auto-generated by adding a line in the *internal/services/firewall/resourceids.go* and run `go generate`, whilst I didn't find that change in this PR. Is that something missed?